extensions/amp-sidebar/1.0/amp-sidebar.js

```diff
@@ -230,6 +244,24 @@ export class AmpSidebar extends AMP.BaseElement {
     this.setupGestures_(this.element);
   }
 
+  /**
+   * Loads the extension for drilldown menu if sidebar contains one and it
+   * has not been installed already.
+   */
+  maybeBuildDrilldownMenu_() {
+    if (this.drilldownMenu_) {
+      return;
+    }
+    const drilldownMenu = this.element.querySelector('amp-drilldown');
+    if (drilldownMenu) {
+      Services.extensionsFor(this.win).installExtensionForDoc(
+        this.getAmpDoc(),
+        'amp-drilldown'
+      );
+      this.drilldownMenu_ = drilldownMenu;
```

Comment:
I'm assuming that the intention here is that only one drilldown menu will exist in `amp-sidebar`. If yes, we should also add this to validation.

Reply:
That's right; I've added some validation items in the design doc under the current approach.